test/paper-button.html

```diff
@@ -13,100 +13,70 @@
-        setTimeout(function() {
-          var shadow = b1.shadowRoot.querySelector('paper-shadow');
-          assert.ok(shadow);
-          done();
-        }, 600);
+      setup(function() {
+        button = fixture('TrivialButton');
       });
-    });
 
-    test('can set noink dynamically', function(done) {
-      var button = document.getElementById('button2');
-      button.lastEvent = {x: 100, y: 100};
-      button.$.ripple = {
-        downAction: function() {
-          assert.ok(false);
-        },
-        upAction: function() {
-          assert.ok(false);
-        }
-      };
-      button.setAttribute('noink', '');
-      fake.downOnNode(button);
-      fake.upOnNode(button);
-      // would throw if it tries to ripple
-      setTimeout(done, 10);
-    });
+      test('can be raised imperatively', function(done) {
+        button.raised = true;
 
-    suite('a11y', function() {
+        expect(button.hasAttribute('raised')).to.be.eql(true);
 
-      test('aria role is a button', function() {
-        assert.strictEqual('button', b1.getAttribute('role'));
+        Polymer.Base.async(function() {
+          try {
+            expect(button.elevation).to.be.eql(1);
+            done();
+          } catch (e) {
+            done(e);
+          }
+        }, 1);
       });
 
-      test('aria-disabled is set', function(done) {
-        var button = document.getElementById('disabled');
-        assert.ok(button.hasAttribute('aria-disabled'));
-        button.removeAttribute('disabled');
-        flush(function() {
-          assert.ok(!button.hasAttribute('aria-disabled'));
-          done();
-        });
+      test('has aria role "button"', function() {
+        expect(button.getAttribute('role')).to.be.eql('button');
       });
 
-      test('space triggers the button', function() {
-        var ev = new CustomEvent('keydown', {detail: {key: 'space'}});
-        var sawClick = false;
-        function clickListener() {
-          sawClick = true;
-        }
-        b1.addEventListener('click', clickListener);
-        b1.dispatchEvent(ev);
-        assert.ok(sawClick);
-        b1.removeEventListener(clickListener);
+      test('can be disabled imperatively', function() {
+        button.disabled = true;
+        expect(button.getAttribute('aria-disabled')).to.be.eql('true');
+        expect(button.hasAttribute('disabled')).to.be.eql(true);
       });
 
-      test('enter triggers the button', function() {
-        var ev = new CustomEvent('keydown', {detail: {key: 'enter'}});
-        var sawClick = false;
-        function clickListener() {
-          sawClick = true;
-        }
-        b1.addEventListener('click', clickListener);
-        b1.dispatchEvent(ev);
-        assert.ok(sawClick);
-        b1.removeEventListener(clickListener);
+      test('can be triggered with space', function(done) {
+        button.addEventListener('click', function() {
+          done();
+        });
+        pressSpace(button);
```

Comment:
Yeah my comment offered no good alternative and wasn't too useful because I'm not sure what else we could do. Simple and dumb approach might be to namespace exports to a global of the same name as the file:

```js
TestHelpers = {
  pressSpace: function() { ... },
  pressEnter: function() { ... }
};
```